reviewing.md

```diff
@@ -1,26 +1,26 @@
-## How to review CNF Certification results
+## How to review CNTI Certification results
 
 ### Technical Requirements
 
-1. Verify that the list of files matches the [expected list](https://github.com/cncf/cnf-certification/blob/main/instructions.md#contents-of-the-pr).
-1. Note the vX.Y subdirectory that the PR is in, this is the version of CNF Certification for which certified is being claimed, referenced as the "Certification Version" from hereon.
-1. Verify that the Certification Version is the current or previous version of CNF Certification.
-1. Look at results log, cnf-testsuite-results-YYYY-MMDD-HHMMSS-NNN.yml. Verify that the major.minor component of the CNF Certification match the certification version being claimed. The patch version does not matter.
+1. Verify that the list of files matches the [expected list](https://github.com/lfn-cnti/certification/blob/main/instructions.md#contents-of-the-pr).
+1. Note the vX.Y subdirectory that the PR is in, this is the version of CNTI Certification for which certified is being claimed, referenced as the "Certification Version" from hereon.
+1. Verify that the Certification Version is the current or previous version of CNTI Certification.
+1. Look at results log, cnf-testsuite-results-YYYY-MMDD-HHMMSS-NNN.yml. Verify that the major.minor component of the CNTI Certification matches the certification version being claimed. The patch version does not matter.
 1. Verify that the RESULTS SUMMARY section of results log "X of Y essential tests passed". There should be at least 10 essential tests passed.
```

Comment:
FYI - this sentence "There should be at least 10 essential tests passed" will be updated in the next 1-2 weeks when the v2.0 certification list of tests is updated.